src/main/java/userReg/UserRegUI.java

```diff
@@ -0,0 +1,237 @@
+        selfIntro.setBounds(140, 260, 180, 125);
+        frame.add(selfIntro);
+
+        //Set submit button
+        JButton buttonregister = new JButton("Submit");
+        buttonregister.setBounds(160, 450, 100, 25);
+        frame.add(buttonregister);
+
+        // Set register button and process further action including passing data to UserRegUIController.
+        buttonregister.addActionListener(new ActionListener() {
+            @Override
+            public void actionPerformed(ActionEvent e) {
+
+                String name_v=name.getText();
+                String eMail_v = new String(eMail.getText());
+                String password_v = new String(password.getText());
+                System.out.println(genderText.getText());
+                Gender gender= Gender.valueOf(genderText.getText());
+                int age_v=Integer.valueOf(("".equals(age.getText())?"0":age.getText()));
+                float height_v=Float.valueOf("".equals(height.getText())?"0":height.getText());
+                String programOfStudy_v=programOfStudy.getText();
+                Hobbies hobby_v=Hobbies.valueOf(hobby.getSelectedItem().toString());
+                String selfIntro_v=selfIntro.getText();
+                Gender interestedIn_v=Gender.valueOf(interestedIn.getText());
+                // Initialize
+                User user=new User(name_v, gender, age_v, height_v, programOfStudy_v,
+                        hobby_v, selfIntro_v, interestedIn_v);
+                UserAccount ua= new UserAccount(eMail_v,password_v,user);
+                UserRegUIOutputBoundary presenter = new UserRegUIPresenter();
+                UserRegUIInputBoundary interactor = new UserRegUIInteractor(presenter, users);
+                UserRegUIController userRegUIController=new UserRegUIController(interactor);
+                UserRegUIResponseModel userRegUIResponseModel=userRegUIController.userProfileCreate(ua);
+                UserRegUIResponseModel  prepareView=presenter.prepareView(userRegUIResponseModel);
+                if (prepareView.getRegMessage()!=""){
+                    JOptionPane.showMessageDialog(frame,userRegUIResponseModel.getRegMessage());
+                }
+
+
+            }
+        });
+        frame.setDefaultCloseOperation(JFrame.EXIT_ON_CLOSE);
+        frame.setVisible(true);
+
+    }
+
+    public static void main(String[] args) {
+
+        UserRepoManager users;
+        try{
+            File csvFile = new File("src/main/resources/userReg.csv");
+            users = new UserRepo(csvFile);
+        }catch (IOException e) {
+            throw new RuntimeException("Could not create file.");
+        }
+
+        UserRegUIOutputBoundary presenter = new UserRegUIPresenter();
+        UserRegUIInputBoundary interactor = new UserRegUIInteractor(presenter, users);
+        UserRegUIController userRegUIController = new UserRegUIController(interactor);
+
+        UserRegUI ur=new UserRegUI( users);
+
+
+    }
```

Comment:
We probably need to move this main part out later.  But we'll fix this when more features are merged to the main branch